backend/src/GraphQl/MethodAsService/FileInputData.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+
+namespace Database.GraphQl.MethodAsService;
+
+public sealed record FileInputData(
+    Data Data
+);
+
+public sealed record Data(
+    ComponentCharacteristics ComponentCharacteristics,
+    IReadOnlyList<DataPoint> DataPoints
+);
+
+public sealed record ComponentCharacteristics(
+    string Surface,
+    string Symmetries
+);
+
+public sealed record DataPoint(
+    Incidence Incidence,
+    Emergence Emergence,
+    Results Results
+);
+
+public sealed record Incidence(
+    Wavelengths Wavelengths,
+    Direction Direction
+);
+
+public sealed record Emergence(
+    Direction Direction
+);
+
+public sealed record Results(
+    double Transmittance,
+    double? Reflectance = null,
+    ColorTransmittance? ColorTransmittance = null,
+    ColorReflectance? ColorReflectance = null
+);
+
+public sealed record Direction(
+    int Polar
```

Comment:
Directions are more complex: https://github.com/building-envelope-data/api/blob/48fca7aef7cf813d4a63b7a8497409448fff6406/schemas/opticalData.json#L253-L308 And `Polar` is according to https://github.com/building-envelope-data/api/blob/48fca7aef7cf813d4a63b7a8497409448fff6406/schemas/number.json#L261-L285 either of type double or an object itself with a confidence interval, a uncertainty number and a value.